downstream/modules/automation-hub/proc-pushing-container-images-from-your-local.adoc

```diff
@@ -0,0 +1,60 @@
+
+[id="pushing-container-images-from-your-local]
+
+= Pushing container images from your local
+
+.Procedure
+. From a terminal, log into podman, or any container client currently in use.
++
+----
+> podman pull <__container-name__>
+----
++
+. Once the image is pulled, add tags:
++
```

Comment:
Suggest changing Once to either After or When. 
IBM Style says: Use "Once" only to mean "one time".